data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/client/CloudWatchLogsClientFactory.java

```diff
@@ -20,6 +19,9 @@
  * CloudWatchLogs services.
  */
 public final class CloudWatchLogsClientFactory {
+    private CloudWatchLogsClientFactory() {
+        throw new IllegalStateException("Static Factory Class!");
```

Comment:
You probably don't need this exception.

Reply:
I have removed the exception for the next revision.